pkgs/development/python-modules/anyqt/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "AnyQt";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-S5IouRiibfL9zGHxNdtUitPDA25bx+Z7qBR+C2aDdjs=";
+  };
+
+  doCheck = false; # TODO: xvfb stuff
```

Comment:
Are all tests based in `xvfb`? If not, can you enable tests and explicitly disable the tests that depend on `xvfb` instead?

Reply:
I know those were broken and didn't worked on them yet